modules/imgcodecs/src/grfmt_webp.hpp

```diff
@@ -47,6 +47,10 @@
 
 #ifdef HAVE_WEBP
 
+#include <webp/decode.h>
```

Comment:
Do not include webp here, just forward declare `WebPAnimEncoder` and `WebPAnimDecoder` (`struct WebPAnimEncoder;`)